modules/checklist/steps/set-fonts-and-colors.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Elementor\Modules\Checklist\Steps;
+
+use Elementor\Core\DocumentTypes\Page;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Set_Fonts_And_Colors extends Step_Base {
+	const STEP_ID = 'set_fonts_and_colors';
+
+	public function get_id() : string {
+		return self::STEP_ID;
+	}
+
+	public function is_absolute_completed() : bool {
+		$settings = $this->kit_adapter->get_kit_settings();
+		//var_dump($kit);
+		//var_dump($kit->get_settings());
+		$custom_color = $settings['custom_colors'] ?? '';
+		$custom_fonts = $settings['custom_typography'] ?? '';
+
+		return ! empty( $custom_color ) && ! empty( $custom_fonts );
+	}
+
+	public function get_title() : string {
+		return __( 'Set up your Global Fonts & Colors', 'elementor' );
+	}
+
+	public function get_description() : string {
+		return esc_html__( 'Global colors and fonts ensure a cohesive look across your site. Start by defining one color and one font.', 'elementor' );
+	}
+
+	public function get_cta_text() : string {
+		return esc_html__( 'Go to Site Identity', 'elementor' );
+	}
+
+	public function get_cta_url() : string {
+		$link = Page::get_site_settings_url_config( true );
+
+		if ( ! $link ) {
+			return Page::get_elementor_create_new_page_url();
+		}
+
+		$parsed_url = wp_parse_url( $link );
+		$query_params = [];
+		wp_parse_str( $parsed_url['query'] ?? '', $query_params );
+		$additional_params = [
+			'active-document' => Plugin::$instance->kits_manager->get_active_id(),
+		];
+
+		$merged_params = array_merge( $query_params, $additional_params );
+
+		if ( $this->page_exists( $merged_params ) ) {
+			return $this->wordpress_adapter->add_query_arg( $additional_params, $link );
+		}
+
+		return Page::get_elementor_create_new_page_url();
+	}
```

Comment:
Feels like I've seen this code before. Is it duplicate?

Reply:
No, i removed it from the other step in a previous pr.

Reply:
Not really, is a simillar way of doing things but it is different.